bedrock/firefox/templates/firefox/includes/schemaorg-breadcrump-new-en-us.html

```diff
@@ -0,0 +1,27 @@
+<script type="application/ld+json">
```

Comment:
If you use the new `{% block structured_data %}` block then you can remove the `<script>` tags here and convert this file to `.json`